src/index.js

```diff
@@ -6,41 +6,42 @@ const tags = tagsW3.map(tag => tag.replace(/[<|>]/g, ""));
 
 // Wait for the DOM to be fully loaded
 document.addEventListener("DOMContentLoaded", function () {
-
   uploadTagsFromLocalStorage(); // list all tags in localStorage
   updateProgressBar();
-  // Find the form element and add a submit event listener
-  const form = document.querySelector("form");
-  form.addEventListener("submit", function (event) {
-    // on submit
-    event.preventDefault(); // Prevent the default form submission
-    submitVal(); // call our submitVal function
-  });
+});
+
+// Find the form element and add a submit event listener
+const form = document.querySelector("form");
+form.addEventListener("submit", function (event) {
+  // on submit
+  event.preventDefault(); // Prevent the default form submission
+  submitVal(); // call our submitVal function
+});
+
+function submitVal() {
+  let val = document.getElementById("input").value;
+  // remove all white spaces and non-alphabetical characters except . ! - 
+  let regex = /[^a-zA-Z0-9!.\-]/g;
```

Comment:
I made a mistake with the recent merge re: alphabetical sort. Line 24 is from a previous merge and it should be already part of the `main` branch, but my merge replace it with the older version. 

I made the mistake of merging from GitHub, but I should have done it from VSCode like I always do. It highlights the differences in a way that they are easy to see. 

So just pointing out that this line should override the current regex in main. So sorry for the extra worry. There already plenty of merge conflicts here. Happy to help with the merge (if you still trust me after today :rolling_on_the_floor_laughing:)

Reply:
Let's see if I get it right when merging. 🙊